RELEASES.md

```diff
@@ -205,3 +205,93 @@ To create a minor release:
+are added to the code base. The Tendermint team instead directs all attention to
+ensuring that the existing code is stable and reliable. Broken tests are fixed,
+flakey-tests are remedied, end-to-end test failures are thoroughly diagnosed and
+all efforts of the team are aimed at improving the quality of the code. During
+this period, the upgrade harness tests are run repeatedly and a variety of
+in-house testnets are run to ensure Tendermint functions at the scale it will
+be used by application developers and node operators.
+
+### Nightly End-To-End Tests
+
+The Tendermint team maintains a set of end-to-end tests that run each night
+on the latest commit of the project. During the feature freeze, these tests are
+run nightly and must pass consistently for a release of Tendermint to be
+considered stable.
+
+### Upgrade Harness
+
+The Tendermint team is in the process of creating an upgrade harness. This
+harness will test the process of stopping an instance of Tendermint running one
+version of the software and starting up the same application running the next
+version of Tendermint. This harness will be paired with the ability to crash the
+Tendermint process at a series of pre-defined places in the execution to allow
+the team to ensure that Tendermint successfully upgrades in many
+different start-stop combinations.
+
+### Large Scale Testnets
+
+The Tendermint end-to-end tests run Tendermint networks with a small set of nodes
+numbering in the low dozens at most. Real world deployments of Tendermint often
+have over a hundred nodes just in the validator set, with many others acting as full
+nodes and sentry nodes.
+
+Each test network is run on a set of Digital Ocean virtual machines (VMs). Each
+VM is equipped with 4 Gigabytes of RAM, 2 CPU cores, and 80 Gigabytes of NVMe
+SSD storage. The network runs a very simple key-value store application. During
+each test net, the following metrics are monitored and collected on each node:
+* Tendermint Rounds per height
+* Peers connected
+* Memory resident set size
+* CPU utilization
+* Blocks produced per minute
+* Seconds for each step of consensus (Propose, Prevote, Precommit, Commit)
+* Latency to receive each block proposal
+
+VMs with low-end specifications are used on purpose. Many issues of resource
+contention that real-world deployments of Tendermint will see would not surface
+in our test application otherwise. To remedy this, we use produce
+a resource-constrained environment for testing Tendermint by running it on
+machines with small numbers of CPU cores and limited memory.
+
+#### 200 Node Testnet
+
+To test the stability and performance of Tendermint in a real world scenario,
+a 200 node test network is run. The network comprises 5 seed nodes, 100
+validators and 95 non-validating full nodes. All nodes begin by dialing
+a subset of the seed nodes to discover peers. The network is run for several
+days, with metrics being collected continuously.
```

Comment:
We should also look to involve light clients in some capacity

Reply:
That's a good point. Perhaps there isn't really much advantage beyond the e2e tests